src/@ionic-native/plugins/zeroconf/index.ts

```diff
@@ -0,0 +1,132 @@
+import { Injectable } from '@angular/core';
+import { Cordova, Plugin } from '@ionic-native/core';
+import { Observable } from 'rxjs/Observable';
+
+export interface ZeroconfService {
```

Comment:
Sorry, forget about the changes regarding the capitalization of the name.

I assumed `ZeroConf` is a brand name since the plugin repo has it written in that way. I traced this thing down to this page: http://www.zeroconf.org/ and realized it's called `Zeroconf`.

I will delete the other comments below.